x-pack/plugins/data_visualizer/public/application/file_data_visualizer/components/results_view/results_view.tsx

```diff
@@ -84,7 +84,7 @@ export const ResultsView: FC<Props> = ({
       </EuiFlexGroup>
 
       <EuiSpacer size="m" />
-      <div className="results">
+      <div>
```

Comment:
With the remove of this `div`s class, I imagine you can also remove the `div` itself, as it appears to be extraneous at the moment.